src/zap/zapheaders.cpp

```diff
@@ -282,6 +282,7 @@ void ZapDebugDirectory::SaveOriginalDebugDirectoryEntry(ZapWriter *pZapWriter)
 
 void ZapDebugDirectory::SaveNGenDebugDirectoryEntry(ZapWriter *pZapWriter)
 {
+#if !defined(NO_NGENPDB)
```

Comment:
NO_NGENPDB is used in quite a few places (zapper.h, crossgen.cpp, compile.cpp, zapper.cpp, etc) and I would like to minimize the changes.